tests/lib/rules/lines-around-comment.js

```diff
@@ -89,6 +89,201 @@ eslintTester.addRuleTest("lib/rules/lines-around-comment", {
+            options: [{
+                beforeLineComment: true,
+                allowBlockStart: true
+            }]
+        },
+        {
+            code: "if(true){\n\n// line at block start\nvar g = 1;\n}",
+            options: [{
+                beforeLineComment: true,
+                allowBlockStart: true
+            }]
+        },
+        {
+            code: "function foo(){   \n/* block comment at block start */\nvar g = 1;\n}",
+            options: [{
+                allowBlockStart: true
+            }]
+        },
+        {
+            code: "function foo(){/* block comment at block start */\nvar g = 1;\n}",
+            options: [{
+                allowBlockStart: true
+            }]
+        },
+        {
+            code: "var foo = function(){\n/* block comment at block start */\nvar g = 1;\n}",
+            options: [{
+                allowBlockStart: true
+            }]
+        },
+        {
+            code: "if(true){\n/* block comment at block start */\nvar g = 1;\n}",
+            options: [{
+                allowBlockStart: true
+            }]
+        },
+        {
+            code: "if(true){\n\n/* block comment at block start */\nvar g = 1;\n}",
+            options: [{
+                allowBlockStart: true
+            }]
+        },
+        {
+            code: "while(true){\n\n/* \nblock comment at block start\n */\nvar g = 1;\n}",
+            options: [{
+                allowBlockStart: true
+            }]
+        },
+
+        // check for block end comments
+        {
+            code: "function foo(){\nvar g = 91;\n// line at block end\n}",
+            options: [{
+                afterLineComment: true,
+                allowBlockEnd: true
+            }]
+        },
+        {
+            code: "function foo(){\nvar g = 61;\n\n\n// line at block end\n}",
+            options: [{
+                afterLineComment: true,
+                allowBlockEnd: true
+            }]
+        },
+        {
+            code: "var foo = function(){\nvar g = 1;\n\n\n// line at block end\n}",
+            options: [{
+                afterLineComment: true,
+                allowBlockEnd: true
+            }]
+        },
+        {
+            code: "if(true){\nvar g = 1;\n// line at block end\n}",
+            options: [{
+                afterLineComment: true,
+                allowBlockEnd: true
+            }]
+        },
+        {
+            code: "if(true){\nvar g = 1;\n\n// line at block end\n}",
+            options: [{
+                afterLineComment: true,
+                allowBlockEnd: true
+            }]
+        },
+        {
+            code: "while(true){\n// line at block start and end\n}",
+            options: [{
+                afterLineComment: true,
+                allowBlockEnd: true
+            }]
+        },
+        {
+            code: "while(true){\n// line at block start and end\n}",
```

Comment:
Edit: I just realized it's tested at https://github.com/eslint/eslint/pull/2741/files#diff-4ae76f7151e0ca78d5c85e750abfd765R227 So I imagine the current test can be removed altogether then!